added fourth point

KendallDoesCoding opened the pull request.

ThioJoe commented:
Eh this isn't really necessary, I added warnings within the program when someone tries to scan other people's channels. Also as a side note, the reason for this is that it's simply impossible to delete other's video's comments, not because I implemented any kind of abuse safeguard.

KendallDoesCoding commented:
> Eh this isn't really necessary, I added warnings within the program when someone tries to scan other people's channels. Also as a side note, the reason for this is that it's simply impossible to delete other's video's comments, not because I implemented any kind of abuse safeguard.

Ok! Makes sense!
